api4/channel_test.go

```diff
@@ -1680,12 +1680,8 @@ func TestGetChannelMembers(t *testing.T) {
 		CheckBadRequestStatus(t, resp)
 
 		_, resp = client.GetChannelMembers("", 0, 60, "")
-		// NOTE: for some reason, while using the LocalClient, the route /channels//members is not handled
```

Comment:
Same for `/channels//posts`, which is why I needed this PR for `getPostsForChannel`

There are some "hidden" dependencies in our routes

Reply:
Yep, it definitely looks like it. If this happens again and knowing this now, what we can do is directly migrate all the subrouters to `LocalRouter` and then build the API on top of them, instead of migrating the ones we need for each PR. Plenty are migrated already, so hope this is the last we see of the issue